### DISCUSSION TITLE
Questions about transfer protein and enzyme settings

### DISCUSSION THREAD

**Discussion by SCZ1174** [Open-Systems-Pharmacology/Forum#1426] [Questions & Problems]
Hello everyone, I have a few problems based on the following article.
Yeung CHT, Beers JL, Jackson KD, Edginton AN. Verifying in vitro-determined enzyme contributions to cannabidiol clearance for exposure predictions in human through physiologically based pharmacokinetic modelling. CPT Pharmacometrics Syst Pharmacol. 2022 Dec 20. doi: 10.1002/psp4.12908. Epub ahead of print. PMID: 36540909.
1. Efflux biliary, how is the transporter protein set up at the apical of the liver?
2. Can the enzyme contribution to clearance be seen in the software or incorporated manually?
![74280c1b3aa686ad1418a12c43093d4](https://user-images.githubusercontent.com/86552551/210385919-9dabd802-f47d-48f5-8c7a-48636e473f0f.png)
![7f2f467aef1e622084725de51a021a9](https://user-images.githubusercontent.com/86552551/210385928-8524e12b-c8ef-453f-8b46-d934cb1bb111.png)

**Comment by StephanSchaller** [comment]
Hi @SCZ1174 , 

1. Which version are you working in? In the expression profile, you can select "apical" for your transporter at the respective organ where it is expressed (dropdown menu)
2. THis can not be set upfront, but you will have to compare the "fraction metabolized" after the simulation
